developer/src/common/web/utils/src/types/ldml-keyboard/ldml-keyboard-xml-reader.ts

```diff
@@ -19,7 +19,10 @@ interface NameAndProps  {
 };
 
 export class LDMLKeyboardXMLSourceFileReaderOptions {
+  /** path to the CLDR imports */
   importsPath: string;
```

Comment:
Although this is a change to the compiler API, we can do that for major releases, and given `localImportsPath` is going to be needed anyway, it doesn't really hurt (plus, I don't really think anyone else is using this API surface outside Keyman yet anyway)

Reply:
I looked for a way to avoid localImportsPath but we just don't have the information at the right point. It does make the Reader object less reusable (usable for one file - basically, one use)